source/jsonnet/england-wales/ccs/blocks/who-lives-here/anyone_else_list_collector.jsonnet

```diff
@@ -70,7 +70,7 @@ local editQuestion(questionTitle) = {
         guidance: {
           contents: [
             {
-              description: 'Remember to only include those people who share cooking facilities and share a living room or sitting room or dining area',
+              description: 'Remember to only include those people who share cooking facilities <strong>and</strong> share a living room, <strong>or</strong> sitting room, <strong>or</strong> dining area.<p><strong>Anyone else is not part of your household and will be interviewed separately.</strong>',
             },
```

Comment:
Instead of using `p` tags, we should use another `description` element.
```js
            {
              description: 'Remember to only include those people who share cooking facilities <strong>and</strong> share a living room, <strong>or</strong> sitting room, <strong>or</strong> dining area.',
            },
            {
              description: '<strong>Anyone else is not part of your household and will be interviewed separately.</strong>',
            },
```